pkg/cloudprovider/provider/gce/cloudconfig_test.go

```diff
@@ -0,0 +1,53 @@
+//
+// Google Cloud Provider for the Machine Controller
+//
+// Unit Tests
+//
+
+package gce
+
+//-----
+// Imports
+//-----
+
+import (
+	"testing"
+)
+
+//-----
+// Tests
+//-----
+
+func TestCloudConfigAsString(t *testing.T) {
+	tests := []struct {
+		name     string
+		config   *cloudConfig
+		contents string
+	}{
+		{
+			name: "minimum test",
+			config: &cloudConfig{
+				Global: global{
+					ProjectID: "my-project-id",
+					LocalZone: "my-zone",
+				},
+			},
+			contents: "[global]\n" +
```

Comment:
Please use a multi line string using backticks. It improves readability

Reply:
Got a different opinion when the rest of the construct, like here, is nested and indented. But it's okay.

Reply:
Good point, feel free to keep it like it is :)